plugins/catalog/src/components/CatalogTable/columns.tsx

```diff
@@ -180,6 +180,16 @@ export const columnFactories = Object.freeze({
       searchable: true,
     };
   },
+  createDisplayNameColumn(options?: {
```

Comment:
what's the purpose of adding this column? 🤔

Reply:
@vinzscam from what I saw the filtering is made by two different things : backend filter (fixed with the lowercasing in the SQL query) and "frontend" filter (native filtering from the table). If I fix only one of them it doesn't work as expected

I tried again to remove it in case I missed something but it really doesn't work without this, it's related to the search feature of `@material-table`

Reply:
mmm I think we have an issue as the `@material-table` search shouldn't be used at all when using pagination

Reply:
@vinzscam So what do we do ? Do you want to check on your side or open an issue about it ? Because for now with my PR it works and fixes a bug so maybe the issue with material-table could be fixed in a separate PR 🙂